ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/node.schema.yml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  node:
+    # Not the most optimal ref, but something fails in schema tooling with recursive refs if we use "eos_designs#/$defs/node_type/keys/nodes/items"
+    $ref: "eos_designs#/$defs/node_type/keys/defaults"
+    type: dict
+    description: |-
+      This key should _only_ be set in hostvars since it is applied unconditionally and overrides any other setting under the node type keys like `l3leaf` or `spine`.
+      It is useful when generating device setting from other systems and sources, since the values can be set with a static path per device instead of having to insert it into the common `<node_type_key>` data structure.
```

Comment:
```suggestion
      It is useful when generating device settings from another system of record since the values can be set with a static path per device instead of inserting it into the common `<node_type_key>` data structure.
```